spec/models/manageiq/providers/ibm_terraform/configuration_manager/refresher_spec.rb

```diff
@@ -89,6 +90,15 @@ def assert_specific_aws_configured_system(configuration_profile_id, orchestratio
       expect(aws_configured_system.counterpart).to eq(cross_link_aws_vm)
     end
 
+    def assert_aws_configured_system_hostname_with_multiple_tags
+      aws_configured_system = ems.configured_systems.find_by(:manager_ref => "5fb677b9ae457000181dd463")
+      expect(aws_configured_system).to have_attributes(
+        :type                     => "ManageIQ::Providers::IbmTerraform::ConfigurationManager::ConfiguredSystem",
+        :hostname                 => "agostino-hybrid-1",
```

Comment:
You shouldn't need this much leading whitespace, just enough to line it up with `:hostname =>`

Reply:
thanks, will fix it

Reply:
I pushed a fix. Thanks for the review!